### PR TITLE
[#645] CodeView: improve UI of sort bar

### LINKED ISSUE #645: Improve the UI of the sort bar in CodeView
Mentioned in https://github.com/reposense/RepoSense/pull/445#pullrequestreview-216125097.

image

* Let's move the sorting to the top, leaving the radio controls at the bottom to emphasize the number of radio buttons better.
* The gap between the sorting and radio controls need to be wider, preferably same distance like the 2 radio button divs.
* The font sizes of 3 controls are different (search bar, checkbox, sort); should make it the same.
* The font reverse checkbox should be black to differentiate it from other checkboxes and to be in consistent with chart view.

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #645 

```
In the UI of sortbar: 
- The font size of 3 controls (searchbar, checkbox, sort) are different
- The sorting option's position is below the radio controls
- The font of reverse checkbox is red in colour
- The gaps between sorting option and radio controls is not 
  standardised

Lets:
- Standardise the font size of all 3 controls so that the sort bar looks 
  neater
- Shift sorting option to be above radio controls to further emphasise
  the number of radio buttons better
- Set the font of reverse checkbox to be black in color to differentiate
  from other checkboxes and to be consistent with the ChartView
- Standardise the gaps between sorting option and radio controls
```

### CODE REVIEW

File: frontend/src/static/css/style.scss [hunk 1]
```diff
@@ -316,12 +316,12 @@ header{
         }
 
         .contribution{
-            margin-top:0.1rem;
             color:mui-color("red");
+            font-size: 0.875rem;
```
Comment (fzdy1914) [inline frontend/src/static/css/style.scss]: Seems unnecessary. But it is OK to leave it here.

File: frontend/src/index.pug [hunk 2]
```diff
@@ -221,6 +221,17 @@ html
...
+              .mui-select
+                select(v-model="filesSortType")
+                  option(value="lineOfCode") LoC
+                  option(value="path") Path
+                  option(value="fileName") File Name
+                  option(value="fileType") File Format
+                label sort by
+              label
+                input(type="checkbox", v-model="toReverseSortFiles").mui-checkbox
+                span#reverse reverse
```
Comment (fzdy1914) [inline frontend/src/index.pug]: Why `#reverse` but not `.reverse` ?

File: frontend/src/static/css/style.scss (hunk 1 above)
Comment (jamessspanggg) [reply frontend/src/static/css/style.scss]: Yep it is redundant, will remove this line. Thank you.

File: frontend/src/static/css/style.scss [hunk 3]
```diff
@@ -346,8 +345,8 @@ header{
 
             #search{
                 padding:0.5rem 1.0rem 0.25rem 1.0rem;
-                margin-top:1.0rem;
-                width:30%;
+                margin-top:1.25rem;
+                width: 30%;
```
Comment (eugenepeh) [inline frontend/src/static/css/style.scss]: unnecessary change

File: frontend/src/static/css/style.scss [hunk 4]
```diff
@@ -408,6 +409,14 @@ header{
                     margin-left:0.25rem;
                 }
             }
+
+            .reverse {
+                color:mui-color("black");
+            }
+
+            select {
+                font-size:0.875rem;
```
Comment (eugenepeh) [inline frontend/src/static/css/style.scss]: instead of doing this, perhaps specify our own font class and assign it to the elements?

**Comment by jamessspanggg** [comment]
Ready to merge?
